spec/controllers/tt/api/v1/split_details_controller_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe Tt::Api::V1::SplitDetailsController do
+  let(:response_json) { JSON.parse(response.body) }
+
+  describe '#show' do
+    it 'returns fake split details' do
+      get :show, format: :json
+      expect(response_json).to eq TestTrack::FakeServer.split_details
```

Comment:
do we want to list out the expected values? this seems like it's just asserting that a variable is equal to itself